samples/besu/simpleasset/contracts/transferInterface.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.8;
```

Comment:
this file should be deleted from here, and probably add this path in `.gitignore` of `simpleasset`, so that it doesn't get checked in by mistake.